code/game/objects/items/manuals.dm

```diff
@@ -172,7 +172,7 @@
 				It can cook multiple items at once.
 
 				<h2>Processor:</h2>
-				Use it to process certain ingredients (meat into faggot, doughslice into spaghetti, potato into fries,etc...)
```

Comment:
even if it is a perfectly acceptable way to refer to a culinary item, it's still safer to remove it in case there's some sort of bot that scrapes for the word regardless of context.

Reply:
Yeah, you need to remove this. It is still a homophobic slur.